compiler/src/Canvas.js

```diff
@@ -46,6 +46,11 @@ function Canvas(id, w, h, wString, hString) {
 
 // add layer to a canvas
 function addLayer(layer) {
+    console.log(
+        `adding new layer in canvas: ${this.id} with id: ${this.layers.length}`
+    );
```

Comment:
I think these log lines might not be necessary.

Reply:
Fixed!